docs/release-procedure.md

```diff
@@ -20,15 +20,16 @@ Release per project:
         python setup.py sdist bdist_wheel --universal
         twine upload dist/*
 
-*   Update conda recipe feedstock on `conda-forge <https://conda-forge.github.io/>`_.
+*   Update conda recipe feedstocks on `conda-forge <https://conda-forge.github.io/>`_.
 
     *  Update conda-smithy and run conda-smithy rerender
 
-            git clone git@github.com:conda-forge/dask-feedstock
-            cd dask-feedstock
+            git clone git@github.com:conda-forge/dask-core-feedstock
+            cd dask-core-feedstock
             conda install conda-smithy
             conda-smithy rerender
 
     *  Get sha256 hash from pypi.org
     *  Update version number and hash in recipe
```

Comment:
Maybe add `* Zero build number`

Reply:
Whoops.  I failed to do this.  My apologies.

Reply:
No worries. More of a suggestion really. We're going to start adding contribution guidelines to the feedstocks, which should have details like this.